### PR TITLE
Add protocol guessing

### PR DESCRIPTION
**Pull request by IzzyDotExe**
Provides a fix for #15 
The game will attempt to add a protocol to urls when not given any. 

Examples:
```
canada.ca -> https://canada.ca
github.com/izzydotexe -> https://github.com/izzydotexe
```

A little more user friendly this way. 

https://github.com/marmust/internet-scanner/assets/44146685/05039217-e99c-40bc-ab4b-1f29e6e7ef5b

### CODE REVIEW

File: Unity_stuff/graph renderer unity project/Assets/TextMesh Pro/Examples & Extras/Resources/Fonts & Materials/Electronic Highway Sign SDF.asset
Comment (IzzyDotExe) [inline Unity_stuff/graph renderer unity project/Assets/TextMesh Pro/Examples & Extras/Resources/Fonts & Materials/Electronic Highway Sign SDF.asset:1]: not sure why it changed this prob some unity housekeeping

**Comment by CADIndie** [comment]
At the moment it seems that your changes assume that all websites use HTTPS, maybe try attempting HTTPS first and then based on the return you attempt HTTP?

**Comment by marmust** [comment]
ok damn this is super nice
tysm this will be very useful :D
ill merge it in a sec

**Comment by Inertia-Squared** [comment]
> At the moment it seems that your changes assume that all websites use HTTPS, maybe try attempting HTTPS first and then based on the return you attempt HTTP? I'm not sure how this program works so I could be suggesting something already done/a non issue so I apologize for the ignorance 😅

If I can find the time this afternoon I'll implement this for them!

**Comment by IzzyDotExe** [comment]
> At the moment it seems that your changes assume that all websites use HTTPS, maybe try attempting HTTPS first and then based on the return you attempt HTTP? I'm not sure how this program works so I could be suggesting something already done/a non issue so I apologize for the ignorance 😅

Nah you are right, i just defaulted to https cause most sites these days use it and i was too lazy to make it try http lol

**Pull request merged**